lib/main_common.pm

```diff
@@ -1093,6 +1093,7 @@ sub load_consoletests {
 
     loadtest "locale/keymap_or_locale";
     loadtest "console/check_upgraded_service" if (is_sle && !get_var('MEDIA_UPGRADE') && !get_var('ZDUP') && is_upgrade && !is_desktop && !get_var('INSTALLONLY'));
+    loadtest "console/supportutils" if (is_sle && !get_var('MEDIA_UPGRADE') && !get_var('ZDUP') && is_upgrade && !is_desktop && !get_var('INSTALLONLY'));
```

Comment:
As you duplicate condition from the line above, what's about extracting it to separate method and explaining what is the rational behind this condition?

Reply:
@rwx788  
The condition is make sure we only add module within migration test case/ only for sle etc..   I do not want side effect for other team. Maybe i also need check with ha/sap team?

Reply:
@juadk could you also have a check?

Reply:
> @rwx788
> The condition is make sure we only add module within migration test case/ only for sle etc.. I do not want side effect for other team. Maybe i also need check with ha/sap team?

That doesn't justify a reason not to extract the condition `is_sle && !get_var('MEDIA_UPGRADE') && !get_var('ZDUP') && is_upgrade && !is_desktop && !get_var('INSTALLONLY')` since we can't easily tell, if `is_online_upgrade_but_not_desktop_and_we_run_more_tests_after_install`

Reply:
Since it's the same condition, you can do as suggested by @rwx788 and @foursixnine and extract `loadtest "console/check_upgraded_service"` and `loadtest "console/supportutils"` to a new method, and then call that method when the condition is true.

Alternatively, you could also do this:

```
if (is_sle && !get_var('MEDIA_UPGRADE') && !get_var('ZDUP') && is_upgrade && !is_desktop && !get_var('INSTALLONLY')) {
    loadtest "console/check_upgraded_service";
    loadtest "console/supportutils";
}
```

That way you avoid repeating the same condition twice.

Reply:
> @juadk could you also have a check?

@coolgw Sorry for late answer, I was on vacations the last 2 weeks.
@alvarocarvajald answered for me, thanks.